holmes.go

```diff
@@ -366,9 +366,9 @@ func (h *Holmes) writeProfileDataToFile(data bytes.Buffer, dumpType configureTyp
 	}
 }
 
-func (h *Holmes) debugUniform(msg string, name string, min, diff, abs int, data []int, cur int) {
-	h.debugf("[Holmes] %v %v, config_min : %v, config_diff : %v, config_abs : %v, previous : %v, current: %v",
-		msg, name, min, diff, abs, data, cur)
+func (h *Holmes) debugUniform(msg string, name string, min, diff, abs, max int, data []int, cur int) {
```

Comment:
could I create a new function type such `uniformFunc func(string,int,int,int,int,[]int,int)` and modify `infoUniform` / `debugUniform` as this new function type? This will restric further change.

Reply:
how to modify this?  show me the pseudo code.

Reply:
meybe we can set a fmt variable, and unified format output?
`h.debugf(fmtvariable, msg, name, min, diff, abs, max, data, cur)`

Reply:
ok, I will add a fmtvariable constant, and could we delete the `debugUniform`/`infoUniform` function, and just using`debugf`/`logf` function?

Reply:
It's ok~